src/routes/business/documents.ts

```diff
@@ -0,0 +1,45 @@
+import type { Request, Response } from "express";
+
+import generateID from "../../helpers/id";
+import * as log from "../../logger";
+
+export const postDocument = async (req: Request, res: Response) => {
+  const documentType = req.body.document_type as string;
+
+  if (!documentType) {
+    return res.status(400).send({
+      id: generateID(),
+      status: 400,
+      code: "bad_request",
+      title: "Bad Request",
+      detail: "document_type is required",
+    });
+  }
+
+  if (!req.file) {
+    return res.status(400).send({
+      id: generateID(),
+      status: 400,
+      code: "bad_request",
+      title: "Bad Request",
+      detail: "file is required",
+    });
+  }
+
+  log.info("Document uploaded", {
+    person_id: req.params.person_id,
+    documentType,
+    file: req.file.originalname,
+  });
+
```

Comment:
would it have sense to track somewhere that the file was uploaded?

Reply:
we receive a document id back, if request was successful
I don't see any benefits in tracking it